internal/fpath/path.go

```diff
@@ -12,7 +12,10 @@ import (
 	"strings"
 )
 
-const storjScheme = "sj"
+var storjScheme = map[string]bool{
```

Comment:
This works, but perhaps you want to make it even better by declaring it as `map[string]struct{}`, as described in this article: https://emersion.fr/blog/2017/sets-in-go/

Reply:
Will look into that!